ddtrace/tracer/transport.go

```diff
@@ -100,9 +102,10 @@ func newHTTPTransport(addr string, client *http.Client) *httpTransport {
 	if cid := internal.ContainerID(); cid != "" {
 		defaultHeaders["Datadog-Container-ID"] = cid
 	}
+	agentURL := getAgentURL(addr)
 	return &httpTransport{
-		traceURL: fmt.Sprintf("http://%s/v0.4/traces", resolveAgentAddr(addr)),
-		statsURL: fmt.Sprintf("http://%s/v0.6/stats", resolveAgentAddr(addr)),
+		traceURL: fmt.Sprintf("%s/v0.4/traces", agentURL),
+		statsURL: fmt.Sprintf("%s/v0.6/stats", agentURL),
```

Comment:
Why did you remove `http://` here to add it 3 times in the below function? I'd add it back and remove it below, avoiding all the `fmt.Sprintf` stuff.

Reply:
This should be resolved in latest changes.